libyul/YulControlFlowGraphExporter.cpp

```diff
@@ -74,9 +74,8 @@ Json YulControlFlowGraphExporter::exportFunction(SSACFG const& _cfg)
 	Json functionJson = Json::object();
 	functionJson["type"] = "Function";
 	functionJson["entry"] = "Block" + std::to_string(_cfg.entry.value);
-	functionJson["arguments"] = Json::array();
-	for (auto const& [arg, valueId]: _cfg.arguments)
-		functionJson["arguments"].emplace_back(arg.get().name.str());
+	static auto constexpr argsTransform = [](auto const& _arg) { return fmt::format("v{}", std::get<1>(_arg).value); };
+	functionJson["arguments"] = _cfg.arguments | ranges::views::transform(argsTransform) | ranges::to<std::vector>;
 	functionJson["returns"] = Json::array();
 	for (auto const& ret: _cfg.returns)
 		functionJson["returns"].emplace_back(ret.get().name.str());
```

Comment:
What about the function returns? I guess it should match the `valueID` in [returnValues](https://github.com/ethereum/solidity/blob/654ce6b36a6ee1d33757584a8c1ae5d9ed51be06/libyul/backends/evm/SSAControlFlowGraph.h#L112) in the exit block no? Maybe we can get it from [functions](https://github.com/ethereum/solidity/blob/654ce6b36a6ee1d33757584a8c1ae5d9ed51be06/libyul/backends/evm/SSAControlFlowGraph.h#L228)

Reply:
good point

Reply:
Yeah, I agree that removing it is a better option. And we need to make Alejandro aware about this change.

Reply:
As a note, we decided to retain the `returns` field because it may provide useful information, even though the same result could be achieved by computing the size of `returnValues` in the exit block. But at least now it doesn't provide misleading information.